x-hooks/use-bounty.tsx

```diff
@@ -223,10 +224,29 @@ export function useBounty() {
       })
   }
 
+  function validateKycSteps(){
+    const sessionSteps = state?.currentUser?.kycSession?.steps;
+    const bountyTierNeeded = state?.currentBounty?.data?.kycTierList;
+    const settingsTierAllowed = state?.Settings?.kyc?.tierList;
+    if(!sessionSteps?.length || !bountyTierNeeded?.length) return;
+
+    const missingSteps = settingsTierAllowed
+                          ?.filter(({id}) => bountyTierNeeded.includes(+id))
+                          ?.map(tier=>({
+                            ...tier,
+                            steps: sessionSteps
+                                    .filter(({id, state}) => tier.steps_id.includes(id) && state !== "VALIDATED")
```

Comment:
you have this tier on an enum somewhere, use that enum instead. if there's no enum, create one and use the enum

Reply:
https://github.com/clarkjoao/web-network/blob/79e9b1c94e43a24e9ae54a8faba4c1760c8c0cf3/interfaces/kyc-session.ts#L2